.github/workflows/ci.yml

```
@@ -1,6 +1,8 @@
 name: CI
 
-on: push
+on: 
+  - push
+  - pull_request
```

To prevent CI running twice for maintainers who push to a branch on the repo 😅 
```suggestion
on:
  push:
    branches:
      - main
  pull_request:
```